code/game/gamemodes/marker/unitologist.dm

```diff
@@ -10,8 +10,8 @@ GLOBAL_LIST_EMPTY(unitologists_list)
 	message_necromorphs("<span class='cult'>[usr]: [message]</span>")
 */
 /datum/antagonist/unitologist
-	role_text = "Unitologist"
-	role_text_plural = "Unitologists"
+	role_text = "Zealot"
```

Comment:
Unitologist Zealot instead?

Reply:
that was already done, look at the latest commits